rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRAStatusHolder.java

```diff
@@ -48,6 +48,7 @@
     String[] failedParticipants;
 
     private LRAStatus lraStatus;
+    private String lraStatusReason;
```

Comment:
Rename: `lraStatusReason` to `statusReason` since the field is already part of an LRA data structure.